marklogic-data-hub-central/ui-custom/src/components/SocialMedia/SocialMedia.test.tsx

```diff
@@ -0,0 +1,155 @@
+import {render} from "@testing-library/react";
+import {DetailContext} from "../../store/DetailContext";
+import SocialMedia from "./SocialMedia";
+
+const configMultiple = {
+  component: "SocialMedia",
+  config: {
+    id: "socialmedia",
+    title: "Social Media",
+    site: {
+      arrayPath: "result[0].extracted.person.socials.social",
+      path: "site"
+    },
+    url: {
+      arrayPath: "result[0].extracted.person.socials.social",
+      path: "address"
+    },
+    sites: {
+      facebook: {
+        title: "facebook",
+        icon: "Facebook",
+        color: "#3B5998"
+      },
+      twitter: {
+        title: "twitter",
+        icon: "Twitter",
+        color: "#00ACEE"
+      },
+      linkedin: {
+        title: "linkedin",
+        icon: "Linkedin",
+        color: "#0E76A8"
+      },
+      instagram: {
+        title: "instagram",
+        icon: "Instagram",
+        color: "#8134AF"
+      },
+      youtube: {
+        title: "youtube",
+        icon: "Youtube",
+        color: "#c4302b"
+      }
+    }
+  }
+}
+const configSingular = {
+  component: "SocialMedia",
+  config: {
+    id: "socialmedia",
+    title: "Social Media",
+    site: {
+      arrayPath: "result[0].extracted.person.social",
+      path: "site"
+    },
+    url: {
+      arrayPath: "result[0].extracted.person.social",
+      path: "address"
+    },
+    sites: {
+      facebook: {
```

Comment:
I think we want the size optionally configurable. They can add a "size" property in any of the "sites" objects, but if the size is not there, we offer a reasonable default.

Reply:
@wooldridge  I did the check inside the widget if the size doesn't come from the config I used 18 by default.
![image](https://user-images.githubusercontent.com/88790827/156640074-63502242-d513-453c-aae0-70febc41e203.png)